safe_transaction_service/contracts/management/commands/update_safe_contracts_logo.py

```diff
@@ -0,0 +1,81 @@
+from django.core.files import File
+from django.core.management import BaseCommand, CommandError
+
+from gnosis.eth import EthereumClientProvider
+from gnosis.safe.safe_deployments import safe_deployments
+
+from safe_transaction_service.contracts.models import Contract
+
+
+def get_deployment_addresses(safe_deployments: dict, chain_id: str) -> list:
+    """
+    Return the deployment addresses of passed dict for a chain id.
+    :param safe_deployments:
+    :param chain_id:
+    :return:
+    """
+    addresses = []
+    if isinstance(safe_deployments, dict):
+        for key, value in safe_deployments.items():
+            if isinstance(value, dict):
+                addresses.extend(get_deployment_addresses(value, chain_id))
+            elif key == chain_id:
+                addresses.append(value)
+    return addresses
+
+
+class Command(BaseCommand):
+    help = "Update safe contract logos by new one"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--safe-version", type=str, help="Contract version", required=False
+        )
+        parser.add_argument(
+            "--logo-path", type=str, help="Path of new logo", required=True
+        )
+
+    def handle(self, *args, **options):
+        """
+        Command to add or update safe contract logos if exist.
```

Comment:
if they exist 😉

Reply:
Please refresh, I omit the exist word because is inherent to update action. https://github.com/safe-global/safe-transaction-service/pull/1657/files#diff-5406ab57a21f508ae31ed44d888934b809b7799f9d45083780326197c25e5f81R38